ProcessManager.py

```diff
@@ -42,11 +57,20 @@ def startAllProcesses(self):
         TODO: create a function that starts all processes or threads in "self.T" all at once.
         """
 
+        for process in process_list:
+            process_list[process].set = process_list[process].cmd
+
     def stopAllProcesses(self):
         """
         TODO: create a function that stops all processes or threads in "self.T" all at once.
         """
 
+        print ("Shutting down")
+
+        for process in process_list:
+            process.kill()
+            process.wait()
```

Comment:
In this for loop, `process` is a `RoboProcess` which doesn't have methods `kill()` or `wait()`. If `RoboProcess` stores a `Popen` object, you'll have to access that and call its `kill()` and `wait()` methods.